Formula/scala.rb

```diff
@@ -3,46 +3,47 @@ class Scala < Formula
   homepage "https://www.scala-lang.org/"
 
   stable do
-    url "https://downloads.lightbend.com/scala/2.12.6/scala-2.12.6.tgz"
-    mirror "https://downloads.typesafe.com/scala/2.12.6/scala-2.12.6.tgz"
-    mirror "https://www.scala-lang.org/files/archive/scala-2.12.6.tgz"
-    sha256 "1ac7444c5a85ed1ea45db4a268ee9ea43adf80e7f5724222863afb5492883416"
+    url "https://downloads.lightbend.com/scala/2.12.7/scala-2.12.7.tgz"
+    mirror "https://www.scala-lang.org/files/archive/scala-2.12.7.tgz"
+    mirror "https://downloads.typesafe.com/scala/2.12.7/scala-2.12.7.tgz"
+    sha256 "d65b0db501287a0fed5a78d92c37cc558af52d9b5339e74f5dd6c71fb736d184"
 
     depends_on :java => "1.8+"
 
     resource "docs" do
-      url "https://downloads.lightbend.com/scala/2.12.6/scala-docs-2.12.6.txz"
-      mirror "https://www.scala-lang.org/files/archive/scala-docs-2.12.6.txz"
-      sha256 "fbbca33acf7492b08b69eb969ea211186c1d70ad57e4f2a2145a770bd9d9a61f"
+      url "https://downloads.lightbend.com/scala/2.12.7/scala-docs-2.12.7.txz"
+      mirror "https://www.scala-lang.org/files/archive/scala-docs-2.12.7.txz"
+      url "https://downloads.typesafe.com/scala/2.12.7/scala-docs-2.12.7.txz"
```

Comment:
`url` shouldn't be used twice here.

Reply:
Whoops!  I could have sworn I already fixed that…

Rebasing now.